test/controllers/departureP5/IntentionToControlP5ControllerSpec.scala

```diff
@@ -121,9 +121,33 @@ class IntentionToControlP5ControllerSpec extends SpecBase with AppWithDefaultMoc
           val view = injector.instanceOf[IntentionToControlP5View]
 
           contentAsString(result) mustEqual
-            view(intentionToControlP5ViewModel, departureIdP5, customsOfficeContactViewModel)(request, messages).toString
+            view(intentionToControlP5ViewModel, departureIdP5, messageId, customsOfficeContactViewModel)(request, messages).toString
       }
     }
-  }
 
+    "must redirect to Presentation notification frontend" in {
+      forAll(listWithMaxLength[RequestedDocumentType]()) {
+        requestedDocuments =>
+          forAll(arbitrary[CC060CType].map(_.copy(RequestedDocument = requestedDocuments))) {
+            message =>
+              val intentionToControlInformationRequestedController: String =
+                controllers.departureP5.routes.IntentionToControlP5Controller.informationRequested(departureIdP5, messageId).url
+
+              when(mockDepartureP5MessageService.getMessage[CC060CType](any(), any())(any(), any(), any())).thenReturn(Future.successful(message))
+              when(mockDepartureP5MessageService.getDepartureReferenceNumbers(any())(any(), any()))
+                .thenReturn(Future.successful(DepartureReferenceNumbers(lrn.value, None)))
+              when(mockReferenceDataService.getCustomsOffice(any())(any(), any())).thenReturn(Future.successful(Right(customsOffice)))
+              when(mockIntentionToControlP5ViewModelProvider.apply(any())(any()))
+                .thenReturn(IntentionToControlP5ViewModel(sections, requestedDocuments = true, Some(lrn.toString)))
```

Comment:
Are these mocks used?

Reply:
You're right, it is a leftover from copy and pasting.